lib/sidekiq/manager.rb

```diff
@@ -107,7 +107,7 @@ def hard_shutdown
         jobs = cleanup.map { |p| p.job }.compact
 
         logger.warn { "Terminating #{cleanup.size} busy threads" }
-        logger.warn { "Jobs still in progress #{jobs.inspect}" }
```

Comment:
Another unrelated change from main. Were some commits merged or rebased wrong? I believe the change is correct, this log was recently changed to debug on main.

Reply:
Hum, that's weird, I must have messed something up, but no idea what, my last few commits look clean.